staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -79,7 +79,7 @@ type AuditWebhookOptions struct {
 func NewAuditOptions() *AuditOptions {
 	return &AuditOptions{
 		WebhookOptions: AuditWebhookOptions{Mode: pluginwebhook.ModeBatch},
-		LogOptions:     AuditLogOptions{Format: pluginlog.FormatLegacy},
+		LogOptions:     AuditLogOptions{Format: pluginlog.FormatJson},
```

Comment:
I think this will break the e2e test. I filed https://github.com/kubernetes/kubernetes/issues/49653 to migrate the e2e to the json format, but for now I think you need to set the format flag here: https://github.com/kubernetes/kubernetes/blob/04c863cb1d89392dac4e88b86d586236e1c284ab/cluster/gce/gci/configure-helper.sh#L1319-L1321